app/components/Authenticated.js

```diff
@@ -35,7 +36,7 @@ const Authenticated = observer(({ auth, children }: Props) => {
   }
 
   auth.logout(true);
-  return null;
+  return <Redirect to="/" />;
```

Comment:
🍖  Previously we relied on `auth.logout()` to reload the page, at which point the server would see there is no accessToken cookie and render the homepage. Now, we can just send the user to the login screen directly as it's rendered by the same React app.